frontend/src/main/pages/HomePage.js

```diff
@@ -25,6 +26,21 @@ export default function HomePage() {
         url: "/api/commons/all"
       }
     );
+  const onSuccess = (commons) => {
+    // Stryker disable next-line all : hard to get variable
+    var existed = new Boolean(false);
```

Comment:
we should try to stick to newer convention for variable scope which is to use `let` or `const`. could you make this change?
```suggestion
    let existed = new Boolean(false);
```

Reply:
changed